src/App/App.scss

```diff
@@ -1,3 +1,3 @@
-.App {
-  text-align: center;
-}
+// .App {
```

Comment:
delete contents of this file - no commented code

Reply:
DAMMIT! All fixed up.